packages/core/components/FileList/test/LazilyRenderedRow.test.tsx

```diff
@@ -24,20 +25,23 @@ describe("<LazilyRenderedRow />", () => {
         const fileSet = new FileSet();
         sinon.stub(fileSet, "getFileByIndex").callsFake((index) => {
             if (index === 3) {
-                return new FileDetail({
-                    annotations: [
-                        {
-                            name: "someDateAnnotation",
-                            values: ["2019-05-17T07:43:55.205Z"],
-                        },
-                    ],
-                    file_id: "abc123",
-                    file_name: "my_image.czi",
-                    file_path: "some/path/to/my_image.czi",
-                    file_size: 1,
-                    thumbnail: "",
-                    uploaded: new Date().toISOString(),
-                });
+                return new FileDetail(
+                    {
+                        annotations: [
+                            {
+                                name: "someDateAnnotation",
+                                values: ["2019-05-17T07:43:55.205Z"],
```

Comment:
Is supposed to be a date, or a datetime?  I thought I saw something about 'bucketing' these values, and it seems like the precision of a datetime is overkill, and a date would be fine.
I suppose the 'date' bucket might put a load on the cache clearing.  And that this might not be the best place to mention it.